devzair/mail-logger-bundle/1.0/config/packages/mail_logger.yaml

```diff
@@ -0,0 +1,2 @@
+mail_logger:
+    enabled: true

```

Comment:
Would you like to put `enabled: true` in env variable, like in example https://github.com/symfony/recipes-contrib/pull/1766/files ?